README.md

```diff
@@ -47,6 +47,14 @@ docker:
     - "/path/to/another-image-name.tar"
 ```
 
+### Run a Command
+device-init can execute a list of commands on boot. e.g.:
+```yaml
+runcmd:
+  - "apt-get install package-name"
```

Comment:
hmmm...cloud init documents examples without quotes. That's why I added tests for it. On the other hand the internet is full of forum post of users asking for help, because they did not realize that their unquoted commands produces incorrect yaml.
So, to keep things save and simple I would suggest not to document that quotes can be removed in some cases